desugreet/entrance_handler.py

```diff
@@ -0,0 +1,60 @@
+import discord
+
+class EntranceHandler:
```

Comment:
can we rename it to `EntranceService`?

I feel like a `Handler` should have the same methods as the the bot provides such as `on_message` etc.

Reply:
I'll rename it